modules/oadp-creating-restore-hooks.adoc

```diff
@@ -54,22 +53,21 @@ spec:
               - /bin/ash
               - -c
```

Comment:
maybe add under init also the timeout option.
initContainers:
..
..
timeout: ...

Reply:
@mperetzred 
What should it be exactly?

Reply:
please see the definition of the timeout here:
https://pkg.go.dev/github.com/vmware-tanzu/velero/pkg/apis/velero/v1#InitRestoreHook